src/resources/MachineLearning/SESConfiguration/SESConfigurationInterfaces.ts

```diff
@@ -0,0 +1,50 @@
+import {DocumentRequirementStatus} from '../Document.js';
+import {FilterConditions} from '../FilterConditions.js';
+
+export interface SESDocumentGroupPreviewParams {
+    /**
+     * The sources to consider.
+     */
+    sources: string[];
+    /**
+     * An array of filtering conditions.
+     */
+    filterConditions: FilterConditions[];
+    /**
+     * The query that determines the documents to extract
+     */
+    advancedQuery: string;
+}
+
+export interface SESDocumentGroupPreview {
+    /**
+     * The query that was used to fetch document information.
+     *
+     * @Example @source==("Salesforce Notifier")
+     */
+    query: string;
+    /**
+     * The total number of documents in the selected sources.
```

Comment:
That match the query, or all docs?

The same question applies to the other "numberOf..." response body properties.

Reply:
`numberOfDocumentsInSources` that are in the source, irregardless of the query, that's kind of the point, we are providing a breakdownof everything that compose the query. the `numberOfValidDocuments` will match the generated `query` in it's entirety (or the provided query given the case